.solr_wrapper

```diff
@@ -0,0 +1,9 @@
+# Place any default configuration for solr_wrapper here
+# port: 8983
+
+instance_dir: tmp/solr-development
+
+download_dir: tmp/solr_wrapper
+
+version: 6.5.1
```

Comment:
Version 1.1.0 of solr wrapper can take `latest` as a version. This is the default, so it doesn't make sense to change it here.